bin/run-browser-tests-azure

```diff
@@ -4,10 +4,13 @@
 
 import subprocess
 
-
 def get_test_files():
-    test_files = ['src/file.test.ts']  # file upload tests
-    return test_files
+    # Disable Azure browser tests for now. We previously only ran file_upload.test.ts in
+    # Azure (see https://github.com/civiform/civiform/issues/5299), but new file upload
+    # tests aren't passing in Azure. Since there's no CEs planning to deploy with Azure,
+    # it's not worth spending the time to try and fix the new file upload tests and
+    # we'll instead just not run any tests on Azure.
```

Comment:
(blocking )Instead of a comment, have this print to the screen so it shows up if someone runs this..

Reply:
Updated to print to the screen! I tested locally that running `bin/run-browser-tests-azure` will print this warning then proceed to run the `file_upload` browser tests.